go.mod

```diff
@@ -2,6 +2,12 @@ module github.com/vmware-tanzu/nsx-operator
 
 go 1.17
 
+replace (
+	github.com/TaoZou1/vsphere-automation-sdk-go/lib => github.com/TaoZou1/vsphere-automation-sdk-go/lib v0.0.0-20221020005919-3ff6e9703580
+	github.com/TaoZou1/vsphere-automation-sdk-go/runtime => github.com/TaoZou1/vsphere-automation-sdk-go/runtime v0.5.1-0.20221020063209-29af114be4a1
+	github.com/TaoZou1/vsphere-automation-sdk-go/services/nsxt => github.com/TaoZou1/vsphere-automation-sdk-go/services/nsxt v0.0.0-20221020005919-3ff6e9703580
```

Comment:
github.com/vmware/vsphere-automation-sdk-go/services/nsxt ?

Reply:
On main branch is github.com/vmware/vsphere-automation-sdk-go